src/scss/base.scss

```diff
@@ -1,18 +1,241 @@
+//IMPORTS
 @import 'variables/colors';
+@import 'external/bootstrap_override';
 
-body {
-  background-image: url('../assets/wolox_bg.png');
+//MIXINS
+@mixin stopDisplay($max-width) {
+  @media screen and (max-width: $max-width) {
+    display: none;
+  }
 }
 
-.center {
-  background-color: rgba(255, 255, 255, .9);
+body {
+  text-align: center;
   height: 100%;
-  left: 20%;
+  background: url('../assets/slide1.jpg') no-repeat center center fixed;
+  background-size: cover;
+  transition: background-image 0.5s ease;
+}
+
+
+li {
+  list-style-type: none;
+}
+
+
+header {
+  overflow: hidden;
+  background-color: $background;
+
+  //NAVIGATION
+
+  ul {
+    display: inline-block;
+  }
+
+  li {
+    padding: 10px 27% 0 0;
+  }
+
+  a {
+    color: $text-color;
+    transition: color 0.2s ease;
+
+    &:hover {
+      color: $text-hover;
+    }
+  }
+
+  nav {
+    position: relative;
+    padding: 35px 0 20px 0;
+    font-size: 10px;
+    top: 0;
+    text-align: left;
+
+    button {
+      background-color: $transparent;
+      border: 0;
+      width: 15px;
+      outline: none;
+      padding: 0;
+      height: 15px;
+      background-image: url('../assets/menu.png');
+      background-size: contain;
+      background-repeat: no-repeat;
+      display: none;
+      @media screen and (max-width: '800px') {
```

Comment:
add all media queries at the end of the file, this way you can track resolution changes easily and you make sure this media queries always override your previous definitions.